### PR TITLE
Update GBFS feed url for Valence, Fr

### PR DESCRIPTION
**Pull request by adrien-delhorme**

### CODE REVIEW

**Comment by Altonss** [comment]
It is working fine currently, so why would the url need to be changed?

**Comment by adrien-delhorme** [comment]
When I request the `feed_url`, I get:
```
{"success":false,"errorCode":"NOT_FOUND","message":"Introuvable","moreInfo":"No handler found for GET /operation/customer/ube/gbfs/v1/en/","fieldsErrors":null}
```

and I can confirm with the official PBSC App that data on https://www.citybik.es/ are not up to date.

But the new url I provided is no longer working... I have no solution for now.

**Comment by Altonss** [comment]
> and I can confirm with the official PBSC App that data on https://www.citybik.es/ are not up to date.

I don't know why I see this, but on citybik.es the stations of Valence have `  "timestamp": "2022-01-06T10:51:09.424000Z",` as a timestamp. So I thought it is up to date...

**Comment by Altonss** [comment]
This url is working!! https://valence.publicbikesystem.net/ube/gbfs/v1/

File: pybikes/data/gbfs.json [hunk 1]
```diff
@@ -836,7 +836,7 @@
                     "PBSC Urban Solutions"
                 ]
             },
-            "feed_url": "https://valence.publicbikesystem.net/ube/gbfs/v1/en/"
+            "feed_url": "https://valencefr.publicbikesystem.net/ube/gbfs/v1/en/"
```
Comment (Altonss) [inline pybikes/data/gbfs.json:839]:
```suggestion
            "feed_url": "https://valencefr.publicbikesystem.net/ube/gbfs/v1/"
```

**Review by Altonss — CHANGES_REQUESTED** [review]
Here is the review with the updated url, hope the data is up to date!

**Comment by adrien-delhorme** [comment]
You are right, I figured out that PBSC is no longer the official app in Valence, Fr. The data I checked were outdated. There is another official app now, and it is already up to date with Citybik.es... So it seems it is already working with the current feed_url (don't know why).

**Comment by Altonss** [comment]
> You are right, I figured out that PBSC is no longer the official app in Valence, Fr. The data I checked were outdated. There is another official app now, and it is already up to date with Citybik.es... So it seems it is already working with the current feed_url (don't know why).

But somehow I cannot open the link with the /en/, so maybe it would be good to merge this PR with my suggestion!
I see that on https://valence.publicbikesystem.net/ube/gbfs/v1/ the links of the different data are to valencefr, so we should merge this PR :)

**Pull request merged**